Engine/CMakeLists.txt

```diff
@@ -32,6 +33,9 @@ set(SOURCES Fuego/Application.cpp Fuego/Log.cpp Fuego/Layer.cpp
             Fuego/LayerStack.cpp Fuego/ImGui/ImGuiLayer.cpp)
 
 set(DEPENDENCIES ImGui)
+if(WIN32)
```

Comment:
It would be good if we don't have conditions here. You can add a global if which encloses everything in your Windows CMakeLists.txt